earth_enterprise/src/fusion/portableglobe/cutter/cgi-bin/globe_cutter_app.py

```diff
@@ -31,6 +31,7 @@
 import sys
 import time
 import urllib
+from lxml import etree
```

Comment:
This appears to be a new dependency.

Reply:
I agree with @tst-blaird  that a very minor change maybe can still be done on the front end, too, though this may already be caught as part of this backend check using lxml.  It does look like python lxml at least is already packaged in debian, ubuntu, and centos.  This should be confirmed and tested on centos6, though.

Reply:
lxml is used extensively in the repository, so it might be worth making that dependency explicit even if this change is not merged.